manifests/config.pp

```diff
@@ -21,12 +14,12 @@
     owner  => $katello_devel::user,
     group  => $katello_devel::group,
     mode   => '0755',
-  } ->
-  file { "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d/katello.yaml":
-    ensure  => file,
-    content => template('katello/katello.yaml.erb'),
-    owner   => $katello_devel::user,
-    group   => $katello_devel::group,
-    mode    => '0644',
   }
+
+  katello_devel::plugin { 'katello/katello': }
+
+  $extra_plugins.each |String $plugin| {
```

Comment:
Bah, it does work! Flaw in my assumptions if you look at the line I linked, that is trying to deploy the plugins settings file and it is hard coded to katello still. Which, @stbenjam makes generic plugins slightly trickier if they require configuration like Katello currently does.